sdk/synapse/Azure.Analytics.Synapse.AccessControl/src/Generated/RoleAssignmentsClient.cs

```diff
@@ -114,15 +115,15 @@ public RoleAssignmentsClient(Uri endpoint, TokenCredential credential, AccessCon
         /// </code>
         /// 
         /// </remarks>
-        public virtual async Task<Response> CheckPrincipalAccessAsync(RequestContent content, RequestContext context = null)
+        public virtual async Task<Response> CheckPrincipalAccessAsync(RequestContent content, ContentType contentType, RequestContext context = null)
```

Comment:
We will need to update the Synapse CHANGELOG to indicate that this breaking change is being made.

Reply:
Yes, will update after autorest merged.